test/functional/test_framework/wallet.py

```diff
@@ -124,7 +125,16 @@ def rescan_utxos(self):
         res = self._test_node.scantxoutset(action="start", scanobjects=[self.get_descriptor()])
         assert_equal(True, res['success'])
         for utxo in res['unspents']:
-            self._utxos.append(self._create_utxo(txid=utxo["txid"], vout=utxo["vout"], value=utxo["amount"], height=utxo["height"]))
+            # Ignore utxo inside mempool
```

Comment:
Can you explain why? `scantxoutset` doesn't scan the mempool, does it?

Reply:
Without that the test throw txn-mempool-conflict, and get_it cos indeed return the tx that is inside the mempool.